internal/status/ing_status.go

```diff
@@ -359,8 +359,9 @@ func deleteObject(option UpdateOptions, key string, isVSDelete bool, retryNum ..
 	}
 
 	for _, host := range option.ServiceMetadata.HostNames {
-		for i, status := range mIngress.Status.LoadBalancer.Ingress {
-			if status.Hostname != host {
+		for i := 0; i < len(mIngress.Status.LoadBalancer.Ingress); {
```

Comment:
@chauhanshubham This will create problem when in status, there is same host with different IP. I think we should iterate from length to zero index